go-controller/pkg/node/gateway_udn.go

```diff
@@ -81,6 +83,12 @@ type UserDefinedNetworkGateway struct {
 	// reconcile channel to signal reconciliation of the gateway on network
 	// configuration changes
 	reconcile chan struct{}
+
+	// vrfTableId holds the link index of management port interface for this network
+	vrfTableId int
+
+	// gwInterfaceId holds the link index of gateway interface
+	gwInterfaceId int
```

Comment:
nit: `gwInterfaceIndex`

Future work: this could be cached in default network bridgeConfiguration, since it is the same for all